java/Glacier2/greeter/build.gradle.kts

```
@@ -0,0 +1,21 @@
+// Copyright (c) ZeroC, Inc.
+
+plugins {
+    // Register the Ice Builder plugin without applying it here.
```

We need to update this comment here, and in the Greeter demos. To reflect the new Gradle plug-in name.

"Slice Tools for Ice and Java"

What about:

```suggestion
    // Register the Slice Tools for Ice and Java plugin without applying it here.
```